src/workerd/util/strings.h

```diff
@@ -77,10 +77,10 @@ constexpr bool isAlphaLower(const kj::byte c) noexcept {
 }
 
 // Convert ASCII alpha characters in the given string to lowercase in place.
-kj::String toLower(kj::String&& str);
+kj::String toLower(kj::String&& str KJ_LIFETIMEBOUND);
```

Comment:
@danlapid you suggested that adding KJ_LIFETIMEBOUND is sufficient to fix this? There is no compile error or warning with the old code in trace-stream after adding KJ_LIFETIMEBOUND, does the compiler just extend the lifetime on its own?